language/tools/move-package/src/resolution/resolution_graph.rs

```diff
@@ -565,6 +566,29 @@ impl ResolvingGraph {
                             dep_name
                         )
                     })?;
+            } else {
+                // If the current folder exists, do a fetch and reset to ensure that the branch
+                // is up to date
+                // NOTE: this means that you must run the package system with a working network connection
```

Comment:
Might be helpful to make a note of this in the error message for the `git fetch` command